Intersect.Client.Framework/Database/GameDatabase.cs

```diff
@@ -95,6 +103,9 @@ public virtual void SavePreferences()
             SavePreference("NpcOverheadInfo", NpcOverheadInfo.ToString());
             SavePreference("PartyMemberOverheadInfo", PartyMemberOverheadInfo.ToString());
             SavePreference("PlayerOverheadInfo", PlayerOverheadInfo.ToString());
+            SavePreference("UiExpToPercentage", UiExpToPercentage.ToString());
+            SavePreference("UiHpToPercentage", UiHpToPercentage.ToString());
+            SavePreference("UiMpToPercentage", UiMpToPercentage.ToString());
```

Comment:
Use `nameof` instead of a string literal

Reply:
like this? :

![image](https://user-images.githubusercontent.com/17498701/189537274-bdd64dfc-bd03-4cf4-9f5e-49d35836e22e.png)

should i change the rest within `SavePreferences()` as well?